src/libecalc/presentation/yaml/mappers/component_mapper.py

```diff
@@ -241,13 +242,16 @@ def create_model(model: Dict, regularity: Dict[datetime, Expression]):
         name = model.get(EcalcYamlKeywords.name, "")
         user_defined_category = model.get(EcalcYamlKeywords.user_defined_tag, "")
         emission_rate = model.get(EcalcYamlKeywords.installation_venting_emitter_emission_rate)
-
+        emission_rate_type = model.get(EcalcYamlKeywords.venting_emitter_rate_type) or RateType.STREAM_DAY
+        convert_to_stream_day = emission_rate_type == RateType.CALENDAR_DAY
```

Comment:
just do:

`emission_rate.to_stream_day()` ?

Reply:
At this stage, the emission_rate is e.g. a string (refering to e.g. a time series rate in input file). Do you think we should set up and evaluate expressions here, and convert to stream day?

Reply:
This is now done in the `YamlVentingEmitter.get_emission_rate(variables_map)` -> `TimeSeriesStreamDayRate`